src/app/public/modules/file-attachment/file-attachment.service.ts

```diff
@@ -48,20 +48,18 @@ export class SkyFileAttachmentService {
     return fileResults;
   }
 
-  public verifyDropFiles(files: any): boolean {
-    if (files.length > 1) {
-      return false;
-    }
-
+  /**
+   * Returns `true` if a directory is found in the provided `files` parameter.
+   */
+  public hasFolder(files: any): boolean {
```

Comment:
I would rename to `hasDirectory` to match the term used in the comment, and the webkit methods. Also, it would be my preference to use a specific type for the parameter, and then cast to `any` when attempting to call the webkit methods. `const file: any = files.item(index);`

Reply:
I like that better too. Done.